ex3/ex3.js

```diff
@@ -7,3 +7,62 @@
  * containsValue([1,3,5,7,9], 7) === true;
  * containsValue([9,7,5,3,1], 7) === true;
  */
+function containsValue(arrayNumber, searchForNumber) {
+
+	console.assert(isNumber(searchForNumber),'is not a number');
+	//  We define isNumberInArray as false so only if we find the search number we will set it to true 
+	//  otherwise will return falsh
+	var isNumberInArray = false;
+	
+	//  Loop through arrayNumber 
+	for (var i = 0; i < arrayNumber.length; ++i) {
+
+		// Looking for searchForNumber in arrayNumber
+		if(arrayNumber[i] === searchForNumber) {
+			// We found a matching number in the arrayNumber 
+			// 1) so leti's set isNumberInArray to true 
+			// 2) and stop the loop
+			isNumberInArray = true;
+			break;
```

Comment:
Instead of saving the "true" value, breaking, then returning it, you can return it here directly -- it should do exactly the same (you can verify that if you just rerun the tests/asserts in your browser).